benches/bench.rs

```diff
@@ -44,11 +44,11 @@ criterion_group!(
 	config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None))).measurement_time(measurement_time_slow());
 	targets = SyncBencher::http_benches_slow, SyncBencher::websocket_benches_slow
 );
-criterion_group!(
+/*criterion_group!(
```

Comment:
this is just some dummy stuff such that I can bench against the old batch request client.

I will revert this when the PR is reviewed properly.

Reply:
Is it time to uncomment this again before we approve/merge?